rs/dre-canisters/node-provider-rewards/canister/src/lib.rs

```diff
@@ -193,7 +194,7 @@ fn post_upgrade() {
     setup_timers();
 }
 
-#[query(hidden = true, decoding_quota = 10000)]
```

Comment:
Doesn't this mean that, if the candid generator hits it, it will try and add it to the `services` part and fail due to `HttpRequest` and `HttpResponse` not being a Candid type?